examples/basic.rs

```diff
@@ -0,0 +1,15 @@
+use system_tray::client::Client;
+
+#[tokio::main]
+async fn main() {
+    let client = Client::new().await.unwrap();
+    let mut tray_rx = client.subscribe();
+
+    let initial_items = client.items();
+
+    // do something with initial items...
```

Comment:
The build is failing currently because the compiler doesn't like unused. I guess either slapping an `#[ignore_unused]` or just dropping `initial_items` should suffice

```suggestion
    // do something with initial items...
    drop(initial_items);
```